### PR TITLE
Change toString of duration

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: value_wrapper.go [hunk 1]
```diff
@@ -404,8 +404,13 @@ func (valWrap ValueWrapper) String() string {
 		return toWKT(ggval)
 	} else if value.IsSetDuVal() {
 		duval := value.GetDuVal()
-		days := duval.GetSeconds() / (24 * 60 * 60)
-		return fmt.Sprintf("P%vM%vDT%vS", duval.GetMonths(), days, duval.GetSeconds())
+		totalSeconds := duval.GetSeconds() + int64(duval.GetMicroseconds()) / 1000000;
+		remainMicroSeconds := duval.GetMicroseconds() % 1000000
+		fmt.Println(totalSeconds)
+		fmt.Println(remainMicroSeconds)
```
Comment (Nicole00) [inline value_wrapper.go:410]: debug code?

**Comment by Aiee** [comment]
Please remove the debug code

File: value_wrapper.go (hunk 1 above)
Comment (jievince) [reply value_wrapper.go]: Removed.

**Comment by jievince** [comment]
> Please remove the debug code

ok

**Pull request closed without merging**